source/OptimaJet.DWKit.StarterApplication/Forms/Groups/UpdateForm.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using JsonApiDotNetCore.Internal;
+using Microsoft.EntityFrameworkCore;
+using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Repositories;
+using OptimaJet.DWKit.StarterApplication.Services;
+
+namespace OptimaJet.DWKit.StarterApplication.Forms.Groups
+{
+    public class UpdateForm : BaseForm
+    {
+        public UserRepository UserRepository { get; set; }
+        public GroupRepository GroupRepository { get; set; }
+        public ICurrentUserContext CurrentUserContext { get; }
+        public UpdateForm(
+            UserRepository userRepository,
+            GroupRepository groupRepository,
+            ICurrentUserContext currentUserContext)
+        {
+            UserRepository = userRepository;
+            GroupRepository = groupRepository;
+            CurrentUserContext = currentUserContext;
+         }
+        public bool IsValid(int id, Group group)
+        {
+            var retVal = true;
```

Comment:
this is kind of unnecessary abbreviation, yeah? could just be `result`

Reply:
refactored to result